logic/requirements.py

```diff
@@ -299,7 +299,7 @@ def __init__(self, options):
         self.bookshot = AND(FEATHER, HOOKSHOT) # use feather on A, hookshot on B on the same frame to get a speedy hookshot that can be used to clip past blocks
         self.bomb_trigger = BOMB # drop two bombs at the same time to trigger cutscenes or pickup items (can use pits, or screen transitions
         self.shield_bump = SHIELD # use shield to knock back enemies or knock off enemies when used in combination with superjumps
-        self.text_clip = options.nagmessages # trigger a text box on keyblock or rock or obstacle while holding diagonal to clip into the side. Not sure how this works, during testing it looks to be working as intended for logic
+        self.text_clip = False & options.nagmessages # trigger a text box on keyblock or rock or obstacle while holding diagonal to clip into the side. Removed from logic for now
```

Comment:
Removed text clips from logic, commented out the relevant logic in dungeon/overworld as well to not cause confusion